docs/src/test/scala/docs/http/scaladsl/server/directives/PathDirectivesExamplesSpec.scala

```diff
@@ -19,13 +19,13 @@ class PathDirectivesExamplesSpec extends RoutingSpec {
   // matches /foo/
   path("foo"./)
 
-  // matches "/foo/bar"
```

Comment:
good catch! Thanks for changing this. 

I am not very good  at recognizing these type of mistakes and I am working on it.

Reply:
No worries :) Thanks for the PR!